community/examples/hpc-slurm-h4d.yaml

```diff
@@ -76,6 +76,7 @@ deployment_groups:
     source: modules/scripts/startup-script
     settings:
       install_cloud_rdma_drivers: true
+      set_ofi_cloud_rdma_tunables: true
```

Comment:
Did we confirm that the env vars won't negatively effect other workloads?  If so, why not put the logic in the RDMA driver installation script?

Reply:
It shouldn't negatively effect other (RDMA) workloads. If someone decides to use a TCP they'd have to override these variables but in any event they'd probably just set this startup script to false. That leads into why I separated them, I didn't want to couple the installation with the env var setting.

Reply:
That makes sense, thanks for explaining.  Be sure to document that this should be disabled for non-RDMA workloads (probably in the variable description).